.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: ruby
+rvm:
+  - 2.4.1
+before_install:
+    - gem update --system
+    - gem install bundler
+before_script:
+ - chmod +x ./script/cibuild # or do this locally and commit
+
+# Assume bundler is being used, therefore
+# the `install` step will run `bundle install` by default.
+script: ./script/cibuild
```

Comment:
ok this looks like it's building it. but it's not deploying the result automatically. Can we maybe add this?

Reply:
Added deploy script and Github key to travis